packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Live/LiveButton/LiveButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import { useInterval } from 'app/hooks';
+
+import { AnimatedRecordIcon, Button, LoadingDiv } from './elements';
+
+export const LiveButton = ({
+  disable,
+  icon = true,
+  isLoading,
+  message = 'Go Live',
+  onClick,
+}) => {
+  const [hovering, setHovering] = useState(false);
+  const [showIcon, setShowIcon] = useState(icon);
+
+  useInterval(
+    () => {
+      if (hovering) {
+        setShowIcon(!showIcon);
+      }
+    },
+    hovering ? 1000 : null
+  );
+
+  if (!hovering && !showIcon) {
+    setShowIcon(true);
+  }
+
+  const startHovering = () => setHovering(true);
+  const stopHovering = () => setHovering(false);
+
+  if (isLoading) {
+    return <LoadingDiv>Creating Session</LoadingDiv>;
+  }
+
+  return (
+    <Button
+      disable={disable}
+      onMouseEnter={startHovering}
+      onMouseLeave={stopHovering}
+      onClick={onClick}
+    >
+      {/*
+       // @ts-ignore */}
```

Comment:
Do you know why we need this ignore?

Reply:
Yes, it's because react-icons doesn't export proper typescript definitions. At least not at the version we're using.

Granted this probably has more to do with me being a typescript noob than anything else. If you can get TS to shut up about it then let's remove this!

Reply:
ohhh! That explains my TS Ignmore too lol